app/components/Documentation/components/Empty.tsx

```diff
@@ -1,27 +1,33 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import Button from 'common-components/Button';
 import noQuestionsAllQuestionsPage from 'images/noQuestionsAllQuestionsPage.svg?inline';
 
 const Empty: React.FC<{ onClickAddArticle: () => void }> = ({
   onClickAddArticle,
-}): JSX.Element => (
-  <div
-    css={{
-      padding: 24,
-      fontSize: 20,
-    }}
-    className="df aic jcc"
-  >
-    <img src={noQuestionsAllQuestionsPage} className="mr24" />
-    <div>
-      <div className="mb24">
-        {"There aren't any articles yet. Would you like to add your first one?"}
+}): JSX.Element => {
+  const { t } = useTranslation();
+  return (
+    <div
+      css={{
+        padding: 24,
+        fontSize: 20,
+      }}
+      className="df aic jcc"
+    >
+      <img src={noQuestionsAllQuestionsPage} className="mr24" />
+      <div>
+        <div className="mb24">
+          {
+            "There aren't any articles yet. Would you like to add your first one?"
```

Comment:
Should this text also be translated?

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1379/commits/1790d6b63600ce0c570839886c15ec8de32eba8c

Reply:
Confirm